pyscriptjs/src/plugin.ts

```diff
@@ -126,10 +133,26 @@ export type PythonPlugin = {
     configure?: (config: AppConfig) => Promise<void>;
     afterSetup?: (interpreter: InterpreterClient) => Promise<void>;
     afterStartup?: (interpreter: InterpreterClient) => Promise<void>;
-    beforePyScriptExec?: { callKwargs: (options: any) => Promise<void> };
-    afterPyScriptExec?: { callKwargs: (options: any) => Promise<void> };
-    beforePyReplExec?: { callKwargs: (options: any) => Promise<void> };
-    afterPyReplExec?: { callKwargs: (options: any) => Promise<void> };
```

Comment:
Changed `callKwargs` to have actual list of args instead....

Reply:
I would like to make it so that we can use `kwargs`. I'll look into fixing it.